cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereCredentialsStep.tsx

```diff
@@ -223,7 +223,7 @@ export function VsphereCredentialsStep(props: Partial<StepProps>) {
         } else {
             setLoadingDatacenters(false);
         }
-    }, [connected, clearDatacenters]);
+    }, [connected]);
```

Comment:
it may cause a dependency issue if we removed clearDatacenters from the dependency list.

Reply:
Somehow having `clearDatacenters` in the list causes an infinite loop. My best guess is that's because clearDatacenters is actually called in the block, and calling it triggers the block again?